packages/server/src/Hocuspocus.ts

```diff
@@ -372,8 +373,8 @@ export class Hocuspocus {
    * … and if nothings fails it’ll fully establish the connection and
    * load the Document then.
    */
-  handleConnection(incoming: WebSocket, request: IncomingMessage, context: any = null): void {
-    const clientConnection = new ClientConnection(incoming, request, context, this, this.hooks.bind(this), this.debugger, {
+  handleConnection(incoming: WebSocket, request: IncomingMessage): void {
+    const clientConnection = new ClientConnection(incoming, request, this, this.hooks.bind(this), this.debugger, {
```

Comment:
Can we inject `context` into clientConnection.hookPayload.context? Otherwise this would be a BC change if used in Express or another framework